user-access-testing/src/main/java/no/unit/nva/database/LocalIdentityService.java

```diff
@@ -33,10 +33,9 @@
 import software.amazon.awssdk.services.dynamodb.model.TableDescription;
 import software.amazon.awssdk.services.dynamodb.model.TableStatus;
 
-public abstract class LocalIdentityService implements WithEnvironment {
+public class LocalIdentityService implements WithEnvironment {
```

Comment:
not abstract anymore because we want to initialize it in the handler test. We need both a local database for the customer service and for the identity service. We cannot have a common because we have to declare the database key schema. In  the future these two tables should be merged to one.